mds/apis/utils.py

```diff
@@ -37,6 +37,12 @@ class LimitOffsetPagination(pagination.LimitOffsetPagination):
 class MultiSerializerViewSetMixin:
     serializers_mapping = {}
 
+    def perform_create(self, serializer):
+        return serializer.save()
+
+    def perform_update(self, serializer):
+        serializer.save()
```

Comment:
I'd suggest
```suggestion
        return serializer.save()
```
as it enables to easily add a hook on policy save, which is your use case on https://github.com/Polyconseil/plato